src/pascy/l2.py

```diff
@@ -12,15 +13,26 @@ class ArpLayer(Layer):
 
     @staticmethod
     def fields_info():
-        # TODO: Implement this :)
-        pass
+        return [UnsignedShort("hardware_type"),
+                UnsignedShort("protocol_type"),
+                UnsignedByte("hardware_len"),
+                UnsignedByte("protocol_len"),
+                UnsignedShort("operation"),
+                MacAddress("sender_hardware_addr"),
+                IPAddress("sender_protocol_addr"),
+                MacAddress("target_hardware_addr"),
+                IPAddress("target_protocol_addr")]
```

Comment:
It is recommended to add a comma (`,`) at the end of any list item, including the last one.
This way, when someone will add support to a new protocol, the git blame won't be on him for adding the comma, but only for the relevant line